### PR TITLE
Add holocene hardfork docs

### PR DESCRIPTION
**Pull request by boyuan-chen**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Describe what your Pull Request is about in a few sentences.

Add holocene hardfork docs

### CODE REVIEW

File: .gitbook/contents/notices/holocene-breaking-changes.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Prepare for Holocene Breaking Changes
+
+The Holocene upgrade for Boba Sepolia and Boba BNB Testnet will be optimistically activated on **1736150400 Mon Jan 06 08:00:00 UTC 2025**.
+
+## For Node Operators
+
+Node operators are required to upgrade to Fjord before the activation date to avoid chain divergence. For Boba Testnets, the `op-node` release [v1.6.14](https://github.com/bobanetwork/boba/releases/tag/v1.6.14), the `op-erigon` release [v1.2.10](https://github.com/bobanetwork/op-erigon/releases/tag/v1.2.10) and the `op-geth` release [v1.101411.3](https://github.com/bobanetwork/op-geth/releases/tag/v1.101411.3) contain these changes
```
Comment (mmontour1306) [inline .gitbook/contents/notices/holocene-breaking-changes.md]: This line still says "Fjord".

Comment (boyuan-chen) [reply .gitbook/contents/notices/holocene-breaking-changes.md]: Fixed

Comment (boyuan-chen) [reply .gitbook/contents/notices/holocene-breaking-changes.md]: Fixed

Comment (boyuan-chen) [reply .gitbook/contents/notices/holocene-breaking-changes.md]: Fixed

**Review by mmontour1306 — APPROVED** [review]

**Review by jyellick — APPROVED** [review]

**Pull request merged**